ipfs-search/dweb-search-frontend
# merge new functionality

- noDetailPage flag
- nativeIpfs flag
- videolist spacing
To test:
* check video list spacing
* check that using ?...&nativeIpfs=true makes all the gateway links target directly ipfs://<...>
* check that using ?...&noDetailPage=true clicking on a card in search results it jumps directly to the gateway link, rather than detail page
* <optionally> use both flags, from the search page you go directly to ipfs:// 
* Vv.: not using the flags you get normal behavior. Using the flags with 'false' in stead you also get normal behavior.
